components/_patterns/04-templates/full-width.twig

```diff
@@ -0,0 +1,12 @@
+{% embed "@templates/_default.twig" with {
+  menu_items: header.menu_items,
```

Comment:
That works for me too. I had also thought about that. But decided not to just because it would take more time. But if you think it'll be confusing, I'll add another menu set for templates.